sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_access_control_client.py

```diff
@@ -103,7 +103,7 @@ def list_role_assignments(self, role_scope, **kwargs):
         :param role_scope: scope of the role assignments. :class:`KeyVaultRoleScope` defines common broad scopes.
             Specify a narrower scope as a string.
         :type role_scope: str or KeyVaultRoleScope
```

Comment:
To avoid muddying the waters on this PR, cleaning up class references is tracked by #18924